tldraw/tldraw
# [docs] Simplify paths for uncategorised pages

This PR changes the paths that we use for uncategorised pages on the docs site.
It also redirects the old paths to the new ones.

```
// BEFORE
tldraw.dev/docs/ucg/shapes

// AFTER
tldraw.dev/docs/shapes
```

I think that this is more intuitive, and will make the links that we share a bit clearer.
eg: As a by-product, it fixes some broken links from the introduction page.

As a trade-off, this PR makes our pathing system a bit more complicated. But I don't think it's tooooo bad? WDYT?

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Navigate to `/docs/shapes` on the docs site.
2. Make sure that it goes to the Shapes page.
3. Do the same for `/docs/ucg/shapes`


### Release Notes

- Documentation: Cleaned up some paths.
